src/main/java/com/github/miosz/carservice/controller/CarController.java

```diff
@@ -0,0 +1,69 @@
+package com.github.miosz.carservice.controller;
+
+import com.github.miosz.carservice.model.Car;
+import com.github.miosz.carservice.service.CarService;
+import org.springframework.stereotype.Controller;
+import org.springframework.ui.Model;
+import org.springframework.validation.BindingResult;
+import org.springframework.web.bind.annotation.*;
+
+import javax.validation.Valid;
+
+@Controller
+public class CarController {
+
+    final CarService carService;
```

Comment:
Jeśli mamy interfejs w warstwie serwisowej, to powinniśmy zadeklarować interfejs.
```suggestion
    final CarInterface carInterface;
```
chociaż nazwa tego interfejsu może nie jest najlepsza 😉 Jeśli już, to może lepiej interfejs nazwać CarService i jego implementację CarServiceImpl.